cmake/DynamicVersion.cmake

```
@@ -448,6 +458,10 @@ function(get_dynamic_version)
 		string(JSON data SET
 				${data} version \"${CMAKE_MATCH_2}\")
 		file(WRITE ${ARGS_TMP_FOLDER}/.version ${CMAKE_MATCH_2})
+		string(JSON data SET
+				${data} distance \"${CMAKE_MATCH_3}\")
+		string(JSON data SET
+				${data} short_sha \"${CMAKE_MATCH_4}\")
```

```suggestion
				${data} short-hash \"${CMAKE_MATCH_4}\")
```
I think it's more common. Also worth exporting this as well.

---

I've added the export of short hash. It seems redundant with exporting commit, though. I'm afraid I won't have time to work on tests.

---

No worries, I will take over. Thanks for the work :)

> It seems redundant with exporting commit

I've found a use-case because the `PKG-INFO` in a python sdist will only have info of short hash